Hearthstone/app/src/main/java/com/example/hearthstone/ui/searchpage/SearchPageFragment.kt

```diff
@@ -59,8 +59,8 @@ class SearchPageFragment : Fragment() {
             binding.btnSearchSP.visibility = View.VISIBLE
             binding.className.visibility = View.GONE
             binding.backToHome.visibility = View.GONE
-            val nameCard = "Search results for '$it'"
-            binding.textSearch.text = nameCard
+            val text = getString(R.string.search_results_for)
+            binding.textSearch.text = "$text '$it'"
```

Comment:
move this to the viewmodel as livedata and call it through XML text=@{viewmodel.searchText}